var/spack/repos/builtin/packages/siesta/package.py

```diff
@@ -36,6 +41,15 @@ def flag_handler(self, name, flags):
 
     def configure(self, spec, prefix):
         sh = which('sh')
+
+        # Patch config.guess and config.sub from gnu savannah
+        sh("-c", "rm -f Src/config.guess Src/config.sub Src/FoX/config/config.guess Src/FoX/config/config.sub")
+        sh("-c", "wget -O Src/config.guess \"https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD\"")
+        sh("-c", "wget -O Src/config.sub   \"https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD\"")
+
+        sh("-c", "wget -O Src/FoX/config/config.guess \"https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD\"")
+        sh("-c", "wget -O Src/FoX/config/config.sub   \"https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD\"")
```

Comment:
This won't work for users on air-gapped systems. Doesn't the `AutotoolsPackage` base class already have logic for patching `config.guess`?

Reply:
I think this may be solved by subclassing `AutotoolsPackage` instead of `Package`.